app/views/admin/editions/confirm_export.html.erb

```diff
@@ -7,6 +7,9 @@
     <ul>
       <li><%= @filter.page_title %></li>
     </ul>
+    <% if ENV.fetch("GOVUK_WEBSITE_ROOT") =~ /integration|staging/ %>
```

Comment:
This is based on logic in `config/schedule.rb`, but I wonder if there might be a better way to do it?

Reply:
I think this method is okay, but wonder if we can put it somewhere where it can shared across the various places. I wonder if we could put it here for example: https://github.com/alphagov/whitehall/blob/master/lib/whitehall.rb

And then it's `Whitehall.integration_or_staging?` I guess.

Reply:
You could then write a test for it too (although I appreciate it's a very simple regex).